lib/import/BpmnImporter.js

```diff
@@ -172,7 +176,7 @@ BpmnImporter.prototype.add = function(semantic, di, parentElement) {
       // are rendered correctly across different "hacks" people
       // love to model such as cross participant / sub process
       // associations
-      parentElement = null;
+      parentElement = this._canvas.findPlane(parentElement).rootElement;
```

Comment:
I'd love to have just a little more test coverage. This line isn't covered at all. I don't know why this change was necessary.

Reply:
I will add more test coverage for this case. This change is necessary as elements with unset `rootElement` get placed in the currently active plane. As we set the active plane after the import, the Association would be on the `base` plane instead of `rootProcessId`.

Reply:
Got it. 👍🏻